sys-libs/zlib/zlib-1.3.1.recipe

```diff
@@ -44,39 +44,37 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
-	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
-	cmd:grep
 	cmd:ld$secondaryArchSuffix
 	cmd:make
 	cmd:sed
 	"
 
-PATCH()
-{
-	sed -i 's,${CMAKE_INSTALL_PREFIX}/bin',$binDir, CMakeLists.txt
-	sed -i 's,${CMAKE_INSTALL_PREFIX}/include',$includeDir, CMakeLists.txt
-	sed -i 's,${CMAKE_INSTALL_PREFIX}/lib',$libDir, CMakeLists.txt
-	sed -i 's,${CMAKE_INSTALL_PREFIX}/share/man',$manDir, CMakeLists.txt
-	sed -i 's,${CMAKE_INSTALL_PREFIX}/share/pkgconfig',$developLibDir/pkgconfig, \
-		CMakeLists.txt
-}
-
 BUILD()
 {
-	cmake -S. -Bbuild \
-		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
-		-DCMAKE_INSTALL_PREFIX:PATH=$prefix
-	make -C build $jobArgs
+	export mandir=$manDir # ./configure won't accept --mandir, but respects $mandir
+	export CFLAGS="-O2 -g -DNDEBUG" # similar to cmake's "RelWithDebugInfo" as used before.
+
+	./configure \
+		--prefix=$prefix \
+		--libdir=$libDir \
+		--includedir=$includeDir \
+		--sharedlibdir=$libDir
+
+	make $jobArgs
 }
 
 INSTALL()
 {
-	make -C build install
+	make install
 
 	# remove static library
 	rm $libDir/libz.a
 
+	# move zlib.pc to the right place
+	mkdir -p $developDir/lib
+	mv $libDir/pkgconfig $developDir/lib
+
```

Comment:
Can you use `fixPkgConfig` instead? Or does that do something weird again?

Reply:
Also, I think, this should use `$developLibDir` instead of `$developDir/lib` because of secondaryArch.

Reply:
No prepareInstalledDevLib?

Reply:
fixPkgconfig after prepareInstaleledDevelLib :)

Reply:
I don't think the order really matters.

Reply:
Doesn't matter, but common practice.